packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -386,7 +388,7 @@ export class Stepper implements LocalizedComponent, T9nComponent {
 
   private resizeObserver = createObserver(
     "resize",
-    (entries) => (this.elWidth = entries[0].contentRect.width),
+    (entries) => (this.elWidth = entries[0].contentRect.width)
```

Comment:
@anveshmekala can you check your stylelint/prettier config? I think the previous should be correct.

Reply:
Will be handled in followup.